### ISSUE TITLE
Diagram our production process 

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/Story-Engine#35]
I've documented our current process here: 
http://storyengine.io/production-process/

## How do we streamline and simplify?
Goal for January / February: look for ways to 
1) shorten the number of steps in the process. To make it more complex, more efficient.
2) automate or simplify steps in the process. (e.g., using Calend.ly to schedule interviews. what else is like that?) 
3) look for ways to outsource parts of the process. What could a paid intern provide that would free up our time? 
4) train staff to take on parts of the process. 

@prefontaine I think this will be a key objective to accomplish in our work week. 
Basically: at this point we want to design a more streamlined engine. 
To reduce the cost and efforts required, clear production bottlenecks, etc. 

See what you think!

We can continue streamlining / documenting here:
https://public.etherpad-mozilla.org/p/stories

**Comment by OpenMatt** [comment]
Maybe we should start by documenting the steps in the process here:

**Comment by OpenMatt** [comment]
Hey @prefontaine -- if you have a lull in your interview work, maybe we can prioritize this one. I think it'd be useful for us to see all the steps in the process and see if we can streamline. 

We can just write the steps down on our notes pad:
https://public.etherpad-mozilla.org/p/stories

I started by taking a quick and dirty first pass

**Comment by prefontaine** [comment]
Thanks for the great notes. I made a few edits. Let me know what you think. The main place I see where we can streamline is to cut out the final approval of the story. I'm torn on that one though, because it shows a lot of respect for respondents ability to express what they want to say publicly -- and I've gotten good feedback for that.

**Comment by OpenMatt** [comment]
makes sense. let's add to our next weekly. maybe we can hack on it a bit together.

**Comment by OpenMatt** [comment]
This is done for now: https://public.etherpad-mozilla.org/p/stories

Let's revisit in Oct

**Comment by OpenMatt** [comment]
@prefontaine: I agree with your proposal above: we should get approval from the interviewee only *once*, not twice. 

Let's update this document together in the office next week: 
https://public.etherpad-mozilla.org/p/stories

**Comment by prefontaine** [comment]
My rationale for using the edited transcript is that that is the base for
all of the other variations, and what we use as our unit of analysis.
Stories outreach/advocacy purposes would, of course, need to remain
faithful to the meaning in the transcript. One way to address this would be
to make it clear in the consent form. Thoughts?

**Comment by prefontaine** [comment]
Linked to #86
